egs2/aishell4/asr1/local/data.sh

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/env bash
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+. ./db.sh || exit 1;
+
+ . utils/parse_options.sh || exit 1;
+
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+stage=0       # start from 0 if you need to start from data preparation
+stop_stage=100
+
```

Comment:
You should check that the `AISHELL4` is not empty in `db.sh`. Please refer to https://github.com/espnet/espnet/blob/master/egs2/commonvoice/asr1/local/data.sh#L27-L31